src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int lowerBase;
+    private int upperBase;
+    private int side;
+
+    public IsoscelesTrapezoid() {
+        this.side = new Random().nextInt(15);
+        this.upperBase = new Random().nextInt(15);
+        this.lowerBase = new Random().nextInt(15);
```

Comment:
lets init values outside
exact FigureSupplier should generate figures with random properties:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/ce46f7a7-b894-46cb-8d3f-d171679c653c)
```suggestion
```